public/src/components/channelManagement/variantEditorSecondaryCtaEditor.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {
+  createStyles,
+  Theme,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  WithStyles,
+  withStyles,
+} from '@material-ui/core';
+import { Cta, SecondaryCta, SecondaryCtaKind } from './helpers/shared';
+import VariantEditorCtaFieldsEditor from './variantEditorCtaFieldsEditor';
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+const styles = ({ spacing }: Theme) =>
+  createStyles({
+    container: {
+      '& > * + *': {
+        marginTop: spacing(1),
+      },
+    },
+    selectContainer: {
+      height: '50px',
+    },
+    formControl: {
+      minWidth: 240,
+    },
+  });
+
+interface VariantEditorSecondaryCtaEditorProps extends WithStyles<typeof styles> {
+  label: string;
+  cta?: SecondaryCta;
+  updateCta: (updatedCta?: SecondaryCta) => void;
+  onValidationChange: (isValid: boolean) => void;
+  isDisabled: boolean;
+}
+
+const VariantEditorSecondaryCtaEditor: React.FC<VariantEditorSecondaryCtaEditorProps> = ({
+  classes,
+  label,
+  cta,
+  updateCta,
+  onValidationChange,
+  isDisabled,
+}: VariantEditorSecondaryCtaEditorProps) => {
+  const handleChange = (event: React.ChangeEvent<{ value: unknown }>): void => {
+    const value = event.target.value as SecondaryCtaKind | 'None';
```

Comment:
If instead of making `secondaryCta` and optional field on the `Variant` type, we added that as a new `SecondaryCtaKind` i.e.

```ts
export enum SecondaryCtaKind {
  None = 'NoneSecondaryCta',
  Custom = 'CustomSecondaryCta',
  ContributionsReminder = 'ContributionsReminderSecondaryCta',
}
```
it would make the logic generally a little cleaner